src/internal/kopia/upload.go

```diff
@@ -508,13 +513,15 @@ func streamBaseEntries(
 		}
 
 		if err := ctr(ctx, entry); err != nil {
-			return clues.Wrap(err, "executing callback on item").With("item_path", itemPath)
+			return clues.Wrap(err, "executing callback on item").
+				WithClues(ctx).
```

Comment:
double checking because I can't remember what ctr does.  It won't call clues.wrap().WithClues(ctx)?

Reply:
no, we should assume it's an opaque function. It's a function that kopia provides which actually uploads data